Calibration/HcalCalibAlgos/src/RecAnalyzerMinbias.cc

```diff
@@ -96,7 +103,7 @@ RecAnalyzerMinbias::RecAnalyzerMinbias(const edm::ParameterSet& iConfig) {
   // Read correction factors
   std::ifstream infile(cfile);
   if (!infile.is_open()) {
-    theRecalib = false;
+    theRecalib_ = false;
```

Comment:
check if the correction factors are in some of the HCAL condition records, in order to avoid files to be sent around.